cmd/backup/config.go

```diff
@@ -58,6 +61,17 @@ type Config struct {
 	LockTimeout                time.Duration `split_words:"true" default:"60m"`
 }
 
+func (c *Config) resolveSecret(envVar string, secretPath string) (string, error) {
+	if secretPath != "" {
+		data, err := os.ReadFile(secretPath)
+		if err != nil {
+			return "", fmt.Errorf("resolveSecret: error reading secret path: %w", err)
+		}
+		return string (data), nil
```

Comment:
```suggestion
		return string(data), nil
```

Reply:
I was unsure what is preferred, since with the space my coloring showed "string" in blue ( aka type ) and without the space it showed up in orange ( aka function call )

Reply:
`go fmt` will remove it, so I guess that's a reasonable thing to do.